### PR TITLE
HentaiSlayer: translate "alternative name"

### PR DESCRIPTION
**Pull request by cuong-tran**
Checklist:

- [x] Updated `extVersionCode` value in `build.gradle` for individual extensions
- [ ] Updated `overrideVersionCode` or `baseVersionCode` as needed for all multisrc extensions
- [ ] Referenced all related issues in the PR body (e.g. "Closes #xyz")
- [ ] Added the `isNsfw = true` flag in `build.gradle` when appropriate
- [X] Have not changed source names
- [X] Have explicitly kept the `id` if a source's name or language were changed
- [X] Have tested the modifications by compiling and running the extension through Android Studio
- [ ] Have removed `web_hi_res_512.png` when adding a new extension

### CODE REVIEW

File: src/ar/hentaislayer/src/eu/kanade/tachiyomi/extension/ar/hentaislayer/HentaiSlayer.kt [hunk 1]
```diff
@@ -43,27 +42,32 @@ class HentaiSlayer : ParsedHttpSource(), ConfigurableSource {
...
+    override fun popularMangaFromElement(element: Element) =
+        SManga.create().apply {
+            element.selectFirst("div#card-real a")?.run {
+                setUrlWithoutDomain(absUrl("href"))
+                selectFirst("figure")?.run {
+                    selectFirst("img.object-cover")?.run {
+                        thumbnail_url = imgAttr()
+                        title = attr("alt")
+                    }
+                    genre = select("span p.drop-shadow-sm").text()
```
Comment (AwkwardPeak7) [inline src/ar/hentaislayer/src/eu/kanade/tachiyomi/extension/ar/hentaislayer/HentaiSlayer.kt]:
```suggestion
            element.selectFirst("div#card-real a")!!.run {
                setUrlWithoutDomain(absUrl("href"))
                selectFirst("figure")!!.run {
                    selectFirst("img.object-cover")!!.run {
                        thumbnail_url = imgAttr()
                        title = attr("alt")
                    }
                    genre = select("span p.drop-shadow-sm").text()
```
title and url is mandatory and should be populated

File: src/ar/hentaislayer/src/eu/kanade/tachiyomi/extension/ar/hentaislayer/HentaiSlayer.kt [hunk 2]
```diff
@@ -43,27 +42,32 @@ class HentaiSlayer : ParsedHttpSource(), ConfigurableSource {
...
-    override fun popularMangaSelector() = "div > div:has(div#card-real)"
-
-    override fun popularMangaFromElement(element: Element) = SManga.create().apply {
-        element.selectFirst("div#card-real a")?.run {
-            setUrlWithoutDomain(absUrl("href"))
-            selectFirst("figure")?.run {
-                selectFirst("img.object-cover")?.run {
-                    thumbnail_url = imgAttr()
-                    title = attr("alt")
+    override fun popularMangaRequest(page: Int) =
```
Comment (AwkwardPeak7) [inline src/ar/hentaislayer/src/eu/kanade/tachiyomi/extension/ar/hentaislayer/HentaiSlayer.kt]: it is fine to leave them on same line

File: src/ar/hentaislayer/src/eu/kanade/tachiyomi/extension/ar/hentaislayer/HentaiSlayer.kt [hunk 3]
```diff
@@ -97,52 +105,58 @@ class HentaiSlayer : ParsedHttpSource(), ConfigurableSource {
...
-            selectFirst("section > div:nth-child(1) > div:nth-child(2)")?.run {
-                select("h1").text().takeIf { it.isNotEmpty() }?.let {
-                    title = it
+    override fun mangaDetailsParse(document: Document) =
+        SManga.create().apply {
+            document.selectFirst("main section")?.run {
+                selectFirst("img#manga-cover")?.run {
+                    thumbnail_url = imgAttr()
+                    title = attr("alt")
+                }
```
Comment (AwkwardPeak7) [inline src/ar/hentaislayer/src/eu/kanade/tachiyomi/extension/ar/hentaislayer/HentaiSlayer.kt]:
```suggestion
            document.selectFirst("main section")!!.run {
                selectFirst("img#manga-cover")!!.run {
                    thumbnail_url = imgAttr()
                    title = attr("alt")
                }
```

File: src/ar/hentaislayer/src/eu/kanade/tachiyomi/extension/ar/hentaislayer/HentaiSlayer.kt [hunk 4]
```diff
@@ -97,52 +105,58 @@ class HentaiSlayer : ParsedHttpSource(), ConfigurableSource {
...
+                    artist = select("p:has(span:contains(الرسام)) span:nth-child(2)").text()
                 }
-                genre = select("a[href*='?genre=']")
-                    .map { it.text() }
-                    .let {
-                        listOf(genre) + it
+                var desc = "\u061C"
+                selectFirst("section > div:nth-child(1) > div:nth-child(2)")?.run {
+                    select("h1").text().takeIf { it.isNotEmpty() }?.let {
+                        title = it
```
Comment (AwkwardPeak7) [inline src/ar/hentaislayer/src/eu/kanade/tachiyomi/extension/ar/hentaislayer/HentaiSlayer.kt]: should set title in one place. could do `title = <attempt one> ?: <attempt two>!!`

**Review by AwkwardPeak7 — CHANGES_REQUESTED** [review]
rebase on main

File: src/ar/hentaislayer/src/eu/kanade/tachiyomi/extension/ar/hentaislayer/HentaiSlayer.kt (hunk 1 above)
Comment (stevenyomi) [reply src/ar/hentaislayer/src/eu/kanade/tachiyomi/extension/ar/hentaislayer/HentaiSlayer.kt]: And replace `run()` with `with()`.

**Review by stevenyomi — CHANGES_REQUESTED** [review]
Revert all the format changes in your PR.

**Pull request merged**